test/inductor/test_unbacked_symints.py

```diff
@@ -145,7 +148,7 @@ def fn(x):
         expected = fn(*example_inputs)
         torch.testing.assert_close(actual, expected)
 
-    @skipCUDAIf(not HAS_CUDA, "requires cuda")
+    @requires_triton()
```

Comment:
Why not use requires_gpu?

Reply:
I found that some UTs will run on CPU, so I only do the triton judgment.

Reply:
But `requires_triton` seems not coverring all situations, these cases can run on CPU without triton.